src/components/LocationDetails.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactTooltip from "react-tooltip";
+import Collapsible from "react-collapsible";
+
+function LocationDetails(props) {
+  const featureProps = props.feature.properties;
+
+  const propsMapping = [
+    {
+      title: 'Demographic Summary',
+      content: [
+        { id: 'persons_num', label: 'Population' },
+        { id: 'median_aud', label: 'Median Income', format: (val) => `AUD ${val}` },
+      ]
+    },
+    {
+      title: 'Economic Summary',
+      content: [
+        { id: 'quartile', label: 'Income Quartile' },
+        { id: 'inequality', label: 'Inequality (lower is better)', format: (val) => `${val}%` },
+        { id: null, label: 'Visitor time spent by quartile' },
+      ]
+    },
+    {
+      title: 'Growth Summary',
+      content: [
+        { id: 'income_diversity', label: 'GDP Growth Potential', desc: 'Economic growth is an increase in the production of economic goods and services,compared from one period of time to another... Traditionally, aggregate economic growth is measured in terms of gross national product (GNP) or gross domestic product (GDP), although alternative metrics are sometimes used.' },
+        { id: 'bridge_diversity', label: 'Job Resilience', desc: 'The ability to adjust to career change as it happens and,by extension, adapt to what the market demands.' },
+        { id: 'bsns_growth_rate', label: 'Business Growth Index', desc: 'The growth rate is the measure of a company’s increase in revenue and potential to expand over a set period.' },
+        { id: 'SA1_7DIGITCODE_LIST', label: 'Included SA1 Regions' },
+      ]
+    }
+  ];
```

Comment:
I'd like to see the propsMapping object split out into a separate file to make it easier for our data folks to update. Also, it will quickly become large and clutter this component if it remains here.